src/main/java/homework_1/Main.java

```diff
@@ -3,7 +3,17 @@
 public class Main {
 
   public static void main(String[] args) {
-    System.out.println("Hello homework!");
+
+    for (String arg : args) {
+      if (arg.equals("error")) {
+        System.out.println("\u001B[41m" + "Alarm!" + "\u001B[0m");
+        break;
+      }
+      if (arg.length() != 8) {
+        continue;
+      }
```

Comment:
Why do you need this?

Reply:
I think a word needs 8 letters.
I can delete it if necessary.

Reply:
Fix that